package.json

```diff
@@ -2,19 +2,17 @@
   "name": "propsd",
   "version": "3.0.3",
   "description": "A local daemon for loading static and dynamic properties",
-  "main": "lib/index.js",
-  "directories": {
-    "doc": "docs",
-    "bin": "bin"
-  },
   "scripts": {
-    "start": "./bin/server.js",
-    "test": "mocha",
-    "lint": "eslint bin/** lib/** test/**",
-    "coverage": "istanbul cover _mocha -- -R spec",
+    "start": "nodemon ./src/bin/server.js -r babel-require",
```

Comment:
Can we change the command to `nodemon ./src/bin/server.js -r babel-require --` so arguments passed on the command line to `npm run start` are passed to the Propsd server instead of nodemon?